package.json

```diff
@@ -135,7 +135,7 @@
     "tasks:otomi-chart": "node dist/tasks/otomi/otomi-chart.js",
     "tasks:wait-for-dev": "ts-node-dev ./src/tasks/otomi/wait-for.ts",
     "tasks:wait-for": "node dist/tasks/otomi/wait-for.js",
-    "test": "NODE_ENV=test mocha --timeout 25000 -r ts-node/register -r ts-custom-error src/**/*.test.* --exit"
+    "test": "NODE_ENV=test mocha -r ts-node/register -r ts-custom-error --exit src/**/*.test.*"
```

Comment:
The .mocarc.yaml file already require  `-r ts-node/register -r ts-custom-error`. Perhaps it can be removed from here.